.ci/cico_rollout_test.sh

```diff
@@ -114,7 +114,7 @@ oc login "${DEV_CLUSTER_URL}" --insecure-skip-tls-verify \
 echo "setting project"
 oc new-project "${PROJECT}" --display-name="PR ${RH_PULL_REQUEST_ID} - Automated Deployment" > /dev/null 2>&1
 echo "adding roles to project"
-oc policy add-role-to-user edit ibuziuk -n $PROJECT
+oc policy add-role-to-user edit Katka92 ScrewTSW rhopp tomgeorge ibuziuk amisevsk davidfestal skabashnyuk -n $PROJECT
```

Comment:
In general, this code should be refactored and list of users should be defined in a separate file to avoid hardcoding the value

Reply:
I could create an issue for myself and deal with it later

Reply:
up-to-you